docs/getting-started/reading-and-writing.mdx

```diff
@@ -108,4 +108,28 @@ async function getUser(username) {
 
     return result.data;
 }
+```
+
+## Private Mode
+
+Private mode allows a node operator to publicly expose its RPC to the internet without the risk of leaking sensitive data within its database. Clients that wish to read data from the node must authenticate with their keypair, allowing access control logic for data within the database to be enforced. This is opposed to “public mode” (the default), where authentication is not required for reads, and thus any client can claim they are any user.
+
+You can configure your Kwil network to run in private mode.
+
+If your Kwil network is running in private mode, a signer is required for `view` procedures on schemas.
```

Comment:
I like that you have this note here. In other places in the docs, it doesn't clearly make the connection that authenticated call === view. So i would keep this note in your next iteration.